appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -181,11 +182,12 @@ class Appcues internal constructor(koinScope: Scope) {
      *
      * @param type Type of the action that is sent by the experience. ex: "my-trait"
      * @param traitFactory Factory (lambda) responsible for creating the ExperienceTrait registered for given [type]
-     *
+     *                     config is an optional map that will be mapped and used to invoke the custom trait with
```

Comment:
I'm not sure how to best explain both properties for the factory, open for suggestions.